### ISSUE TITLE
Metadata obj inside store value

### ISSUE THREAD

**Issue by sanchis** [AlexMayol/localit#26]
## The objective of the proposal 
To avoid possible name collisions when choosing the key. 
It would be good to store the expiration data (and the necessary ones) in a specific metaData for each case inside the storage.

## Example
```js
store.set('more_info',{ rocketMan: '🚀' }, '2s');
// internally will be saved like this for the key more_info
{ 
    value: { rocketMan: '🚀' }, 
    meta: { created: 1637830075345, expires: 1637830075345 }
}

store.get("more_info");
// This will be return 
{ rocketMan: '🚀' }

```
The keys would be stored exactly the same as the ones entered by the user.

**Comment by AlexMayol** [comment]
There can't be any type of collision when storing the key due to the domain or expiration date because of how's internally being handled, but I like this as a new way of storing the data. It'd remove the need to have `the_key` and `the_key_expiration_date` on the `Storage`.

**Comment by AlexMayol** [comment]
Merged in #29
